### PR TITLE
Changes as proposed in #14

### PR DESCRIPTION
**Pull request by kba**
@Shreeshrii @wrznr

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,15 +3,22 @@ export
 SHELL := /bin/bash
 LOCAL := $(PWD)/usr
 PATH := $(LOCAL)/bin:$(PATH)
-TESSDATA =  $(LOCAL)/share/tessdata 
-LANGDATA = $(PWD)/langdata-$(LANGDATA_VERSION)
+HOME := /home/ubuntu
```
Comment (kba) [inline Makefile]: `$HOME` is set by default (i.e. `HOME := $(HOME)`), if you want to override it you can, but I would not set a different default. As you use it, it seems more like you meant `PWD` or `REPO_ROOT`?

File: Makefile [hunk 2]
```diff
@@ -3,15 +3,22 @@ export
...
+TESSDATA =  $(HOME)/tessdata_best
+LANGDATA = $(HOME)/langdata
```
Comment (kba) [inline Makefile]: I find it preferable to have the explicit version of langdata for reproducible results.

File: Makefile [hunk 3]
```diff
@@ -3,15 +3,22 @@ export
...
 
 # Name of the model to be built
 MODEL_NAME = foo
 
+# Name of the model to continue from
+CONTINUE_FROM = frk
```
Comment (kba) [inline Makefile]: Default to `$(MODEL_NAME)`?

File: Makefile [hunk 4]
```diff
@@ -82,19 +89,22 @@ data/list.eval: $(ALL_LSTMF)
...
+	
+$(TRAIN)/%.box: $(TRAIN)/%.tif $(TRAIN)/%-gt.txt
+	python generate_line_box.py -i "$(TRAIN)/$*.tif" -t "$(TRAIN)/$*-gt.txt" > "$@"
 
 $(ALL_LSTMF): $(sort $(patsubst %.tif,%.lstmf,$(wildcard $(TRAIN)/*.tif)))
 	find $(TRAIN) -name '*.lstmf' -exec echo {} \; | sort -R -o "$@"
 
 $(TRAIN)/%.lstmf: $(TRAIN)/%.box
-	tesseract $(TRAIN)/$*.tif $(TRAIN)/$* lstm.train
+	tesseract $(TRAIN)/$*.tif $(TRAIN)/$*   --psm 6 lstm.train
```
Comment (kba) [inline Makefile]: Make page segmentation mode configurable?

**Comment by Shreeshrii** [comment]
I had changed it for my own testing. Please make the required changes for the generic script.

Some related discussion is at https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/tesseract-ocr/be4-rjvY2tQ/vIF2a0XbCgAJ

**Comment by kba** [comment]
> Hi, this commit changed the gt.txt extension to "-gt.txt" from ".gt.txt". I think it should be set as before.

Thanks for spotting, that was a mistake, fixed in master.

**Pull request merged**